jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -26,6 +27,15 @@
   "The default interval, in seconds, between nemesis operations."
   10)
 
+(defn- followers
+  "Get all nodes which are followers, i.e. not primaries."
+  [db test]
+  (let [primaries (db/primaries db test)
+        nodes (:nodes test)]
+    (filter
+     (fn [node] (not (some (fn [x] (= node x)) primaries)))
+     nodes)))
+
```

Comment:
This probably belongs in jepsen.db, and I'd name it "secondaries", for symmetry with "primaries".

Reply:
Fixed